packages/frontend2/src/server/features/scaling/activity/get-activity-chart.ts

```diff
@@ -4,21 +4,34 @@ import {
   unstable_cache as cache,
   unstable_noStore as noStore,
 } from 'next/cache'
+import { env } from '~/env'
 import { db } from '~/server/database'
+import { getRangeWithMax } from '~/utils/range/range'
+import { generateTimestamps } from '../../utils/generate-timestamps'
 import { getFullySyncedActivityRange } from './utils/get-fully-synced-activity-range'
 import {
   type ActivityProjectFilter,
   createActivityProjectsFilter,
 } from './utils/project-filter-utils'
 import { type ActivityTimeRange } from './utils/range'
 
+/**
+ * Returns a chart data of the activity over time.
+ * @returns [timestamp (number), count (number), ethereumCount (number)]
```

Comment:
It isn't true. The type should be `[timestamp (number), count (number), ethereumCount (number)][]`